lib/Components/Cart/cart.css

```diff
@@ -0,0 +1,211 @@
+    width: 90%;
+}
+
+.sm-cart-products-list .sm-card .sm-card-img {
+    background: var(--COLOR-WHITE-ALWAYS);
+    padding: 3rem;
+}
+
+.sm-cart-products-list .sm-card .sm-card-content {
+    padding: 1rem;
+}
+
+.sm-cart-products-list .sm-card .sm-card-content .sm-card-content-desc {
+    font-size: 1.2rem;
+    padding: 1px;
+}
+
+.sm-card-cart-qty-container {
+    display: flex;
+    align-items: center;
+    gap: 2rem;
+}
+
+.sm-cart-qty {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    font-size: 1.6rem;
+}
+
+.sm-qty-counter-wrapper {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    border-radius: 5px;
+    border: 2px solid var(--COLOR-PRIMARY);
+}
+
+.sm-cart-qty select {
+    border-radius: 5px;
+    padding: 6px;
+    background: var(--COLOR-BG);
+    color: var(--COLOR-FONT);
+    border: 2px solid var(--COLOR-PRIMARY);
+    outline: none;
+    cursor: pointer;
+    font-weight: 800;
+    transition: all ease 0.2s;
+}
+
+.sm-cart-qty select:hover {
+    background: var(--COLOR-PRIMARY);
+}
+
+.qty-btn-red {
+    color: var(--COLOR-FONT);
+    background: rgb(var(--COLOR-RED-1),0.3);
+    border: 2px solid var(--COLOR-RED);
+    border-radius: 5px;
+    padding: 5px 10px;
+    white-space: nowrap;
+    cursor: pointer;
+}
+
+.qty-btn-red:hover {
+    background: rgba(var(--COLOR-RED-1),0.8);
+}
+
+.qty-btn-red:active {
+    transform: scale(0.98);
+}
+
+.sm-cart-checkout-container {
+    display: flex;
+    flex-direction: column;
+    padding: 1.2rem 2rem;
+    border-radius: 5px;
+    border: 1px solid var(--COLOR-PRIMARY);
+    height: fit-content;
+    box-shadow: 0px 3px 2px -2px var(--COLOR-PRIMARY);
+}
+
+.sm-cart-checkout-title {
+    font-weight: 800;
+    font-size: 2.5rem;
+    padding-bottom: 1rem;
+    text-align: center;
+    border-bottom: 1px solid var(--COLOR-PRIMARY);
+}
+
+.sm-cart-checkout-details {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 2rem;
+    font-weight: 500;
+    font-size: 2rem;
+}
+
+.sm-cart-checkout-amt {
+    font-weight: 700;
+}
+
+.sm-cart-total {
+    border-top: 1px solid var(--COLOR-PRIMARY);
+    margin: 1rem 0;
+}
+
+.btn-place-order {
+    font-size: 2rem;
+}
+
+.mt-5 {
+    margin-top: 1rem;
+}
+
+.mr-10 {
+    margin-right: 1rem;
+}
+
+.btn-logged-in-user {
+    position: relative;
+}
+
+.sm-header-nav-user-dropdown {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 2rem;
+    position: absolute;
+    padding: 1rem;
+    border-radius: 10px;
+    top: 7rem;
+    right: 1rem;
+    background: var(--COLOR-BG);
+    box-shadow: 0 0 4px 0 var(--COLOR-BS-GREY);
+}
+
+.user-show-dropdown-hide {
+    display: none;
+}
+
+.btn-logged-in-user:hover + .user-show-dropdown-hide{
+    display: flex;
+}
+
+.user-show-dropdown-hide:hover{
+    display: flex;
+}
+
+/* MAIN ENDS */
+
+/* MEDIA QUERIES */
+@media only screen and (max-width:768px) {
```

Comment:
Suggestion - You can also use variables for breakpoints. That way all the CSS files can be accessed while making changes to the breakpoint. 

For example- 
--BREAKPOINT_MOBILE: 576px;
--BREAKPOINT_TABLET: 768px;
--BREAKPOINT_LAPTOP: 992px;

You can also use- 
--BREAKPOINT_SM: 500px;
--BREAKPOINT_M: 654px;